keyboards/sck/osa/info.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file should be reverted to the version in b3b49c3700151ab8613a50907aa819492099d444.

```
git checkout b3b49c3 -- keyboards/sck/osa/info.json
git commit
```

Reply:
reverted as requested

For my future reference, is configurator not capable of rendering/using the angled alphas?

Reply:
Configurator does not currently support rotated keys. You *could* set up the file so the keys are unrotated but staggered vertically, like the Atreus62 or Ergodox EZ do, but that requires editing the `info.json` by hand for basically every key on every layout. I didn't feel like putting you through that.

Reply:
ok thank you